apps/wallet/src/features/Loyalty/History/index.tsx

```diff
@@ -27,9 +27,9 @@ import ModalHeader from '../components/ModalHeader';
 
 import Item from './Item';
 
-interface PartnerQuestProps {
+type PartnerQuestProps = {
 	safeAreaInsets: EdgeInsets;
-}
+};
```

Comment:
Any specific reason to use `type` instead of `interface`? I think we'll need to clarify distinguish between `interface` and `type` - currently `interface` used quite random by my mistake. What I think, we'll stop using `interface` except polymorphism related places.

Reply:
type is better for typecheck feature with typescript, but they are still overlap in their features. `interface` is only used for OOP inheritant, so I think we should use `type` as rich features of type checking, and `interface` for the case we want to apply OOP (just a convention).